src/brainglobe_napari/napari_atlas_representation.py

```diff
@@ -9,19 +10,40 @@ class NapariAtlasRepresentation:
     """Representation of a BG atlas as napari layers"""
 
     bg_atlas: BrainGlobeAtlas
+    viewer: Viewer
 
-    def add_to_viewer(self, viewer: Viewer):
-        """Adds the annotation and reference images to a viewer.
+    def add_to_viewer(self):
+        """Adds the annotation and reference images,
+        and the top-level "root" mesh, to the viewer, in that order.
 
-        The annotation image is on top, and visible,
-        while the reference image is below it and invisible.
+        The reference image's visibility is off, the others' is on.
         """
-        viewer.add_image(
+        self.viewer.add_image(
             self.bg_atlas.reference,
+            scale=self.bg_atlas.resolution,
             name=f"{self.bg_atlas.atlas_name}_reference",
             visible=False,
         )
-        viewer.add_labels(
+        self.viewer.add_labels(
             self.bg_atlas.annotation,
+            scale=self.bg_atlas.resolution,
             name=f"{self.bg_atlas.atlas_name}_annotation",
         )
+
+        root_mesh = self.bg_atlas.mesh_from_structure("root")
+        self._add_mesh(root_mesh, name=f"{self.bg_atlas.atlas_name}_mesh")
+
+    def _add_mesh(self, mesh: Mesh, name: str = None):
+        """Helper function to add a mesh as a surface layer to the viewer.
+
+        mesh: the mesh to add
+        name: name for the surface layer
+        """
+        points = mesh.points
+        cells = mesh.cells[0].data
+        self.viewer.add_surface(
```

Comment:
Can be addressed later, but can these be defined elsewhere? brainrender has a `settings.py` for example

Reply:
Opened #36 for this.

Reply:
As an initial fix, I've moved the hardcoded values to make them attributes with defaults of the Napari Representation, so they can be set programmatically if needed.